src/org/jetbrains/java/decompiler/main/extern/IFernflowerPreferences.java

```diff
@@ -14,207 +14,258 @@
 public interface IFernflowerPreferences {
   @Name("Remove Bridge Methods")
```

Comment:
The changes made here break any application that passes options to the BaseDecompiler constructor, like the Intellij plugin or Enigma